[2025-01-31T09:58:33.208+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T09:58:33.236+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-python-etl-dag.load scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-01-31T09:58:33.255+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-python-etl-dag.load scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-01-31T09:58:33.257+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-31T09:58:33.530+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load> on 2025-01-30 00:00:00+00:00
[2025-01-31T09:58:33.541+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=88) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T09:58:33.543+0000] {standard_task_runner.py:72} INFO - Started process 94 to run task
[2025-01-31T09:58:33.546+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my-first-python-etl-dag', 'load', 'scheduled__2025-01-30T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpchyk4v4l']
[2025-01-31T09:58:33.548+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask load
[2025-01-31T09:58:33.639+0000] {task_command.py:467} INFO - Running <TaskInstance: my-first-python-etl-dag.load scheduled__2025-01-30T00:00:00+00:00 [running]> on host 0d287206d23f
[2025-01-31T09:58:33.804+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='olalekanrasaq1331@gmail.com' AIRFLOW_CTX_DAG_OWNER='Akinkunmi' AIRFLOW_CTX_DAG_ID='my-first-python-etl-dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-30T00:00:00+00:00'
[2025-01-31T09:58:33.806+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T09:58:33.807+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T09:58:33.808+0000] {logging_mixin.py:190} INFO - Current task name:load state:running start_date:2025-01-31 09:58:33.238003+00:00
[2025-01-31T09:58:33.809+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag and current dag run status:running
[2025-01-31T09:58:33.810+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T09:58:33.811+0000] {logging_mixin.py:190} INFO - Inside Load
[2025-01-31T09:58:33.812+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-31T09:58:33.829+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T09:58:33.830+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my-first-python-etl-dag, task_id=load, run_id=scheduled__2025-01-30T00:00:00+00:00, execution_date=20250130T000000, start_date=20250131T095833, end_date=20250131T095833
[2025-01-31T09:58:33.848+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T09:58:33.848+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T09:58:33.849+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag queued_at:2025-01-31 09:58:15.887300+00:00
[2025-01-31T09:58:33.850+0000] {logging_mixin.py:190} INFO - Task hostname:0d287206d23f operator:PythonOperator
[2025-01-31T09:58:33.882+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T09:58:33.920+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T09:58:33.924+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
